tests/curators/test_curators_general.py

```diff
@@ -235,3 +239,40 @@ def test_schema_ordered_set(df):
     # clean up
     ln.Schema.filter().delete()
     ln.Feature.filter().delete()
+
+
+def test_schema_no_minimal_set_var():
+    """If minimal_set is False, invalid ensembl gene IDs are allowed."""
+    adata = ln.core.datasets.mini_immuno.get_dataset1(otype="AnnData")
+    adata.var_names = [adata.var_names[0], adata.var_names[1], "NOT_VALID_ENSEMBL"]
+
+    var_schema = ln.Schema(
+        itype=bt.Gene.ensembl_gene_id,
+        minimal_set=False,
+    ).save()
+    schema = ln.Schema(otype="AnnData", slots={"var": var_schema}).save()
```

Comment:
This should be `var.T`, shouldn't it? I think this is the default behavior we typically want, don't we? So that people don't have to deal with messing with different Ensembl versions.

Can you add `minimal_set=True` to `ln.schemas.anndata....`?